service/src/main/java/com/ge/predix/acs/monitoring/DecisionCacheHealthIndicator.java

```diff
@@ -0,0 +1,59 @@
+package com.ge.predix.acs.monitoring;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.actuate.autoconfigure.ConditionalOnEnabledHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.redis.connection.RedisConnection;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.core.RedisConnectionUtils;
+import org.springframework.stereotype.Component;
+import org.springframework.util.Assert;
+
+@Component
+@Profile({ "cloud-redis", "redis" })
+@ConditionalOnEnabledHealthIndicator("decision.cache")
```

Comment:
Setting overall status to DEGRADED might trigger an alarm.  May be it's better to set overall status to UP but redis status DISABLED in case we need to temporarily disable caching but don't want any alarms.